test/metabase/lib/test_util.cljc

```diff
@@ -94,22 +103,23 @@
                                            :breakout     [[:field (meta/id :checkins :user-id) nil]]}}
               ;; this is copied directly from a QP response. NOT CONVERTED TO KEBAB-CASE YET, BECAUSE THIS IS HOW IT
               ;; LOOKS IN LEGACY QUERIES!
-              :result-metadata [{:description       nil
-                                 :semantic_type     :type/FK
-                                 :table_id          (meta/id :checkins)
-                                 :coercion_strategy nil
-                                 :name              "USER_ID"
-                                 :settings          nil
-                                 :source            :breakout
-                                 :field_ref         [:field (meta/id :checkins :user-id) nil]
-                                 :effective_type    :type/Integer
-                                 :nfc_path          nil
-                                 :parent_id         nil
-                                 :id                (meta/id :checkins :user-id)
-                                 :visibility_type   :normal
-                                 :display_name      "User ID"
-                                 :fingerprint       {:global {:distinct-count 15, :nil% 0.0}}
-                                 :base_type         :type/Integer}
+              :result-metadata [{:description        nil
+                                 :semantic_type      :type/FK
+                                 :table_id           (meta/id :checkins)
+                                 :coercion_strategy  nil
+                                 :name               "USER_ID"
+                                 :settings           nil
+                                 :source             :breakout
+                                 :field_ref          [:field (meta/id :checkins :user-id) nil]
+                                 :effective_type     :type/Integer
+                                 :nfc_path           nil
+                                 :parent_id          nil
+                                 :id                 (meta/id :checkins :user-id)
+                                 :fk_target_field_id (meta/id :users :id)
```

Comment:
This was missing, I'm not sure why.

Reply:
I believe this was added due to a the drill thru bugs.
Should we cache an actual qp response so we don't have to keep it in sync?